src/main.cpp

```diff
@@ -943,7 +934,9 @@ int main(int argc, char *argv[]) {
 
   push_time = timer.getTimeMicroseconds() - push_time;
 
-  cout << "Setting range..." << endl;
+  if (benchmark_mode == false) {
+      cout << "Setting range..." << endl;
+  }
```

Comment:
Why is this info not printed in benchmark mode?

Reply:
As one might want to log the console output during a benchmark run, it could lead to unexpected disk startups when the buffer is flushed to disk. This causes strange power spikes. As this output is primarily useful for debugging, I disabled the output in benchmark mode. Do you think it is necessary to output this information in benchmark mode?

Reply:
No, I just wondered why you did these changes. Should we add some kind of debug mode in order to enable output of this kind?

Reply:
I assumed that debug mode is the default, if benchmark mode is not selected. So do we want do have benchmark, default and debug mode? Alternatively, we could implement the possibility to redirect the console (debug) output automatically to a file (maybe even with timestamp).

Reply:
Okay. Maybe we should discuss this somewhere else.